docs/azure_jumpstart_ag/freezer_monitor/_index.md

```diff
@@ -4,4 +4,98 @@ weight: 100
 toc_hide: true
 ---
 
-# Overview
\ No newline at end of file
+# 
+
+## Overview + Diagram (if applicable)
+
+![Applications and technology stack architecture diagram](./img/placeholder.png)
+
+### Data flow
+
+![Data flow diagram](./img/placeholder.png)
+(not sure if separate dfd is needed with the Diagram above...
+)
+
+## Instructions
+
+### Scenario: Identifying the broken freezer
+
+The manager of the Chicago store has reported that food in one of the freezers has not been staying frozen. She has moved the food to the second freezer and called for service, but the technician reported that the freezer seems to be operating within parameters. Your job as the data analyst is to determine if the temperature sensor in the freezer has observed the issue, and provide the data to the store manager.
+
+- Confirm the issue in Azure Data Explorer
+    1. From your own computer (not the Client VM) Open the dashboard named "Freezer Monitoring" in Azure Data Explorer: https://dataexplorer.azure.com/dashboards
+    ![Dashboard showing all freezers](./img/placeholder.png)
+
+    2. The charts are quite busy so let's make it easier to see if there is a problem in Chicago by filtering the dashboard to show only data for the Chicago site.
+        ![Dashboard showing Chicago freezers only](./img/placeholder.png)
+
+    3. The store manager didn't tell you which freezer was having problems, but it's obvious that Freezer-2 is regularly exceeding the safe threshold of 20 degrees. You can send information to the store manager via a screenshot, or you can explain to the manager how she can use Grafana in the store to see the same data.
```

Comment:
Should we mention temperature units? Interpretation varies depending on where the user is coming from.

Reply:
Changed to °F